src/OpenFeatureSDK/OpenFeatureSDK.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggerVer)" />
+    <PackageReference Include="System.Collections.Immutable" Version="6.0.0" />
```

Comment:
Can you move the dep version into [Common.props](https://github.com/open-feature/dotnet-sdk/blob/main/build/Common.props#L22). We probably also want to allow a range of Immutable versions instead of hard locking on 6. This SDK currently supports back to net462

Reply:
Yeah, I can do that.

I can add a range as well. 6 works with 462, but so does 1.x and 7.
https://www.nuget.org/packages/System.Collections.Immutable/6.0.0#supportedframeworks-body-tab

Reply:
I made a range of 1.7.1 inclusive through 7.0.0 exclusive. As 7.0.0 isn't released yet. Not much changes about these collections.

Reply:
Thanks, yea it's more to avoid unintentionally upgrading an older project consuming the openfeature dependency on immutable.